package.json

```diff
@@ -20,6 +20,7 @@
     "@fortawesome/fontawesome-free": "^6.4.0",
     "admin-lte": "^4.0.0-alpha2",
     "bootstrap": "^5.3.0",
-    "jquery": "^3.7.0"
+    "jquery": "^3.7.0",
+    "toastr": "^2.1.4"
```

Comment:
toastr is unmaintained 
![image](https://github.com/Relluem94s/RelluToolbox/assets/6181042/2b189391-5929-4cf6-9a97-aa69074052c1)

```JavaScript
var msg = 'Folivora <script>alert("DangerDan!")</sc' + 'ript>'; 
toastr.success("Exploit <strong>Message</strong> " + msg);

Reply:
We will switch out with bootstraps native toasters :)

Reply:
Changed to own baked Toast. Yummy.